common/remote_object.go

```diff
@@ -189,25 +193,71 @@ func valueFromRemoteObject(ctx context.Context, robj *cdpruntime.RemoteObject) (
 	return k6ext.Runtime(ctx).ToValue(val), err
 }
 
-func handleParseRemoteObjectErr(ctx context.Context, err error, logger *logrus.Entry) {
-	var (
-		ooe *objectOverflowError
-		ope *objectPropertyParseError
-	)
-	var merr *multiError
-	if !errors.As(err, &merr) {
-		// If this panics it's a bug :)
-		k6ext.Panic(ctx, "parsing remote object value: %w", err)
-	}
-	for _, e := range merr.Errors {
-		switch {
-		case errors.As(e, &ooe):
-			logger.Warn(ooe)
-		case errors.As(e, &ope):
-			logger.WithError(ope).Error()
-		default:
-			// If this panics it's a bug :)
-			k6ext.Panic(ctx, "parsing remote object value: %w", e)
+func parseConsoleRemoteObjectPreview(logger *log.Logger, op *cdpruntime.ObjectPreview) string {
+	obj := make(map[string]string)
+	if op.Overflow {
+		logger.Warnf("parseConsoleRemoteObjectPreview", "object is too large and will be parsed partially")
+	}
+
+	for _, p := range op.Properties {
+		val := parseConsoleRemoteObjectValue(logger, p.Type, p.Subtype, p.Value, p.ValuePreview)
+		obj[p.Name] = val
+	}
+
+	bb, err := json.Marshal(obj)
+	if err != nil {
+		logger.Errorf("parseConsoleRemoteObjectPreview", "failed to marshal object to string: %v", err)
+	}
+
+	return string(bb)
+}
+
+//nolint:cyclop
+func parseConsoleRemoteObjectValue(
+	logger *log.Logger,
+	t cdpruntime.Type,
+	st cdpruntime.Subtype,
+	val string,
+	op *cdpruntime.ObjectPreview,
+) string {
+	switch t {
+	case cdpruntime.TypeNumber:
+	case cdpruntime.TypeBoolean:
+	case cdpruntime.TypeSymbol:
+	case cdpruntime.TypeBigint:
```

Comment:
Do we need to handle these?

```suggestion
```

Reply:
I added this fro two reasons:
1. It makes it clear that all the types have been considered.
2. The linter was having an issue with not checking all cases.

Happy to remove them and add a nolint directive. WDYT?

Reply:
Then, it makes sense 👍 Maybe we can make it a single-liner?

Reply:
Maybe add them to the end of the switch also, and with a comment? For me it makes it more intuitive to understand that we are doing nothing for these cases.

Reply:
Resolved in 0139348b022c3d9b8bfaa9b32845fb32c1242c1b